docs/_src/api/api/pydoc-markdown-file-classifier.yml

```diff
@@ -15,4 +15,4 @@ renderer:
   descriptive_module_title: true
   add_method_class_prefix: false
   add_member_class_prefix: false
-  filename: file_converter.md
+  filename: file_classifier.md
```

Comment:
This is probably the wrong place to note this, but actually both `file_converter` and `file_classifier` modules exist, and I just realized in the API docs we only have FileConverter. We should add them both now that we're at it